src/SFA.DAS.AssessorService.Data/CertificateRepository.cs

```diff
@@ -184,6 +187,14 @@ private bool CheckCertificateData(Certificate certificate, string lastName)
             return certificate;
         }
 
+        public async Task<Certificate> GetCertificate(string certificateReference)
+        {
+            var certificate = await
+               _context.Certificates
```

Comment:
Is there a reason why in the BatchLogRepository we got rid of the EF context and replaced with Dapper SQL extensions but here we are keeping the EF Context?

Reply:
The whole application was written as a mixture of Dapper and EF; that's bad enough but having repositories which are themselves a mixture of two data access techniques is awful - If I'm changing a repository and the majority is already Dapper I'm converting to Dapper if its simple enough to do - ones such as certificates and contact which are large and almost entirely EF I'm leaving as EF - it would be a larger Tech Debt to convert them.